low-level/files/opt/seagate/sspl/setup/platform_based_config_update.py

```diff
@@ -53,3 +53,24 @@ def update_sensor_info(config_index, node_type, enclosure_type):
         Conf.set(config_index, '%s>monitor' % sect, value)
 
     Conf.save(config_index)
+
+def update_monitored_services(config_index, node_type):
+    """Based on node type exclude the services from list
+    of monitored services.
+    """
+    # Align to node type as used in sspl.conf SERVICEMONITOR section
+    vm_types = ["virtual", "vm"]
+    node_type = "vm" if node_type.lower() in vm_types else "hw"
+
+    monitored_services = Conf.get(
+        config_index, "SERVICEMONITOR>monitored_services", [])
+    excluded_services = Conf.get(
+        config_index,
+        "SERVICEMONITOR>excluded_services>%s" % node_type, [])
+
+    # Have only required list of services to be monitored
+    monitored_services = list(set(monitored_services) - set(excluded_services))
+    Conf.set(
```

Comment:
why saving in config again. Run time we can exclude the services & use it !?

Also this will create confusion that SSPL only monitors this updated service list, if we refer to this sspl.conf file after deployment

Reply:
If we need to do that only during run time without affecting sspl.conf, then in below three files we willbe having duplicate code.

1. service_monitor.py
2. service_msg_handler.py
3. sspl_test.py

I will revert the current changes and address your comment.

Reply:
keep that common code in a function and call wherever required

Reply:
Addressed,
Done.